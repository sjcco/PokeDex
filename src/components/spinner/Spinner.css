.spinner-cont {
  width: 100%;
  display: flex;
  justify-content: center;
  align-items: center;
  height: 100vh;
}

.spinner {
  width: 100px !important;
  height: 100px !important;
}
