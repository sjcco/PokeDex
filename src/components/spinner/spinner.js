import React from 'react';

import CircularProgress from '@material-ui/core/CircularProgress';

import { spinner, spinnerCont } from './Spinner.css';

const Spinner = () => (
  <div className={spinnerCont}>
    <CircularProgress className={spinner} />
  </div>
);

export default Spinner;
