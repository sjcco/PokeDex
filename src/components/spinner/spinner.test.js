/**
 * @jest-environment jsdom
 */

import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import spinner from './spinner';

test('renders content', () => {
  const component = renderer.create(<spinner />).toJSON();
  expect(component).toMatchSnapshot();
});
