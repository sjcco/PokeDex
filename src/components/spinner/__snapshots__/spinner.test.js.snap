// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders content 1`] = `<spinner />`;
